rca/people/tests/test_forms.py

```diff
@@ -1,105 +0,0 @@
-                "relatedlinks-MIN_NUM_FORMS": "0",
-                "relatedlinks-MAX_NUM_FORMS": "5",
-                "related_area_of_expertise-TOTAL_FORMS": "0",
-                "related_area_of_expertise-INITIAL_FORMS": "0",
-                "related_area_of_expertise-MIN_NUM_FORMS": "0",
-                "related_area_of_expertise-MAX_NUM_FORMS": "1000",
-                "related_research_centre_pages-TOTAL_FORMS": "0",
-                "related_research_centre_pages-INITIAL_FORMS": "0",
-                "related_research_centre_pages-MIN_NUM_FORMS": "0",
-                "related_research_centre_pages-MAX_NUM_FORMS": "1000",
-                "related_schools-TOTAL_FORMS": "0",
-                "related_schools-INITIAL_FORMS": "0",
-                "related_schools-MIN_NUM_FORMS": "0",
-                "related_schools-MAX_NUM_FORMS": "1000",
-                "personal_links-TOTAL_FORMS": "0",
-                "personal_links-INITIAL_FORMS": "0",
-                "personal_links-MIN_NUM_FORMS": "0",
-                "personal_links-MAX_NUM_FORMS": "5",
-                "slug": "student-monty-python",
-                "student_user_account": "",
-            }
-        )
-
-    def get_form(self, instance, data=None):
-        return self.form_class(instance=instance, data=data)
-
-    def test_the_valid_form(self):
-        form = self.get_form(instance=self.student_page, data=self.get_form_data(),)
-        self.assertTrue(form.is_valid())
-
-    def test_first_name_required(self):
-        data = self.get_form_data()
-        data["first_name"] = ""
-        form = self.get_form(instance=self.student_page, data=data,)
-        self.assertFalse(form.is_valid())
-        self.assertIn("This field is required", str(form.errors.get("first_name")))
-
-    def test_last_name_required(self):
-        data = self.get_form_data()
-        data["last_name"] = ""
-        form = self.get_form(instance=self.student_page, data=data,)
-        self.assertFalse(form.is_valid())
-        self.assertIn("This field is required", str(form.errors.get("last_name")))
-
-    def test_adding_student(self):
-        data = self.get_form_data()
-        data["student_user_account"] = self.student.id
-        form = self.get_form(instance=self.student_page, data=data)
-        self.assertTrue(form.is_valid())
-
-    def test_adding_student_with_page(self):
-        # If a student user account has already been related to a page, we shouldn't be able
-        # to add them to another
-        page = StudentPageFactory()
-        page.student_user_account = self.student
-        page.save()
-
-        data = self.get_form_data()
-        data["student_user_account"] = self.student.id
-        form = self.get_form(instance=self.student_page, data=data,)
-        self.assertFalse(form.is_valid())
-        self.assertIn(
-            "The Student you have selected already has a user account",
-            str(form.errors.get("student_user_account")),
-        )
```

Comment:
I've had to remove these tests. Validation on the actual BE works, I just cannot seem to test it. The form is always valid. I think it's due to the introduction of the [permission based panels](https://github.com/torchbox/rca-wagtail-2019/pull/630/files#diff-2d3f90b2d4c7cc3f332a28b633d184020ad892ae959d11336d4a9d6788699816R71)